core/domain/exp_domain.py

```diff
@@ -2431,8 +2439,141 @@ def _convert_states_v33_dict_to_v34_dict(cls, states_dict):
             states_dict[key] = state_domain.State.convert_html_fields_in_state(
                 state_dict,
                 html_validation_service.add_math_content_to_math_rte_components)
+
         return states_dict
 
+    @classmethod
+    def _convert_states_v34_dict_to_v35_dict(cls, states_dict):
+        """Converts from version 34 to 35. Version 35 adds translation support
+        for interaction customization arguments. This migration converts
+        customization arguments whose schemas have been changed from unicode to
+        SubtitledUnicode or html to SubtitledHtml.
+
+        Args:
+            states_dict: dict. A dict where each key-value pair represents,
+                respectively, a state name and a dict used to initialize a
+                State domain object.
+
+        Returns:
+            dict. The converted states_dict.
+        """
+
+        def convert_ca_html_list_to_subtitled_html_dict_list(
+                ca_value, content_id_prefix, next_content_id_index):
+            """Convert a customization argument value of type html list to
+            list of subtitled html dict.
+
+            Args:
+                ca_value: list(str)|None. Customization argument value.
+                content_id_prefix: str. The content id prefix.
+                next_content_id_index: int. The next content_id index.
+
+            Returns:
+                list(dict). A list of SubtitledHtml dicts.
+            """
+            new_value = []
+            new_content_ids = []
+
+            for value in ca_value:
+                content_id = (
+                    content_id_prefix +
+                    python_utils.UNICODE(next_content_id_index))
+                new_content_ids.append(content_id)
+                new_value.append({
+                    'content_id': content_id,
+                    'html': value
+                })
+                next_content_id_index += 1
+
+            return new_value, new_content_ids
+
+
+        for state_dict in states_dict.values():
+            # Find maximum existing content_id index.
+            max_existing_content_id_index = -1
+            translations_mapping = state_dict[
+                'written_translations']['translations_mapping']
+            for content_id in translations_mapping:
+                content_id_suffix = content_id.split('_')[-1]
+                if content_id_suffix.isdigit():
```

Comment:
If it is not a digit, we are looking at a content_id such as 'default_outocome', in which case this would not affect our calculation on next content id index.